bot/resources/easter/save_the_planet.json

```
@@ -0,0 +1,71 @@
+[
+    {
+        "title": "Save the Planet: Choose Reneweable Energy",
+        "image": {"url": "https://cdn.dnaindia.com/sites/default/files/styles/full/public/2019/07/23/851602-renewable-energy-istock-072419.jpg"},
+        "footer": {"text": "The best thing you can do is sharing this information!"},
```

```suggestion
        "footer": {"text": "Help out by sharing this information!"},
```
This is a purely subjective comment. To me, something like this seems better, since it's shorter and more direct. It's up to you if you want to change it or not.

---

Yeah that sounds better.